cmd/bucket-lifecycle.go

```diff
@@ -144,10 +144,14 @@ func (es *expiryState) enqueueByNewerNoncurrent(bucket string, versions []Object
 
 var globalExpiryState *expiryState
 
+const (
+	ChannelCapacity = 100000
```

Comment:
Shouldn't be exported and should have a descriptive name.

Reply:
Great! I change these name.

`ChannelCapacity ` to `MaximumExpiryTasks`, could describe more clearly and exactly what's this constant value. It means that maximum expiry tasks.
`ReplicationDefault ` to `DefaultReplicationPoolSize`, defines the default size of the replication pool.